extension/packages/nextjs/app/pool/page.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+
+import { type FC, useMemo, useState } from "react";
+import { TransactionData, getPoolServerUrl } from "../create/page";
+import { TransactionItem } from "./_components";
+import { useInterval } from "usehooks-ts";
+import { useChainId } from "wagmi";
+import {
+  useDeployedContractInfo,
+  useScaffoldContract,
+  useScaffoldEventHistory,
+  useScaffoldReadContract,
+} from "~~/hooks/scaffold-eth";
+import { useTargetNetwork } from "~~/hooks/scaffold-eth/useTargetNetwork";
+import { notification } from "~~/utils/scaffold-eth";
+
+const Pool: FC = () => {
+  const [transactions, setTransactions] = useState<TransactionData[]>();
+  // const [subscriptionEventsHashes, setSubscriptionEventsHashes] = useState<`0x${string}`[]>([]);
+  const { targetNetwork } = useTargetNetwork();
+  const poolServerUrl = getPoolServerUrl(targetNetwork.id);
+  const { data: contractInfo } = useDeployedContractInfo("MetaMultiSigWallet");
+  const chainId = useChainId();
+  const { data: nonce } = useScaffoldReadContract({
+    contractName: "MetaMultiSigWallet",
+    functionName: "nonce",
+  });
+
+  const { data: eventsHistory } = useScaffoldEventHistory({
+    contractName: "MetaMultiSigWallet",
+    eventName: "ExecuteTransaction",
+    fromBlock: 0n,
+    watch: true,
+  });
+
+  const { data: metaMultiSigWallet } = useScaffoldContract({
+    contractName: "MetaMultiSigWallet",
+  });
+
+  const historyHashes = useMemo(() => eventsHistory?.map(ev => ev.args.hash) || [], [eventsHistory]);
```

Comment:
also changed this from`ev.log.args.hash` to `ev.args.hash` since `event history hook was [rewritten](https://github.com/scaffold-eth/scaffold-eth-2/pull/869)